packages/frontend/core/src/modules/properties/services/adapter.ts

```diff
@@ -167,211 +164,65 @@ export class WorkspacePropertiesAdapter extends Service {
-  }
-
-  getItemId(item: WorkspaceFavoriteItem) {
-    return FavoriteItemsAdapter.getFavItemKey(item.id, item.type);
-  }
-
-  getItemOrder(item: WorkspaceFavoriteItem) {
-    return item.order;
-  }
-
-  setItemOrder(item: WorkspaceFavoriteItem, order: string) {
-    item.order = order;
-  }
-
-  // read from the workspace meta and migrate to the properties
-  private migrateFavorites() {
-    // only migrate if favorites is empty
-    if (Object.keys(this.favorites ?? {}).length > 0) {
-      return;
-    }
-
-    // old favorited pages
-    const oldFavorites = this.workspace.docCollection.meta.docMetas
-      .filter(meta => meta.favorite)
-      .map(meta => meta.id);
-
-    this.adapter.transact(() => {
-      for (const id of oldFavorites) {
-        this.set(id, 'doc', true);
-      }
-    });
-  }
-
-  isFavorite(id: string, type: WorkspaceFavoriteItem['type']) {
-    const existing = this.getFavoriteItem(id, type);
-    return existing?.value ?? false;
-  }
-
-  isFavorite$(id: string, type: WorkspaceFavoriteItem['type']) {
-    return this.favorites$.map(() => {
-      return this.isFavorite(id, type);
-    });
-  }
-
-  private getFavoriteItem(id: string, type: WorkspaceFavoriteItem['type']) {
-    return this.favorites?.[FavoriteItemsAdapter.getFavItemKey(id, type)];
-  }
-
-  // add or set a new fav item to the list. note the id added with prefix
-  set(
-    id: string,
-    type: WorkspaceFavoriteItem['type'],
-    value: boolean,
-    order?: string
-  ) {
-    this.adapter.ensureRootProperties();
-    if (!this.favorites) {
-      throw new Error('Favorites is not initialized');
-    }
-    const existing = this.getFavoriteItem(id, type);
-    if (!existing) {
-      this.favorites[FavoriteItemsAdapter.getFavItemKey(id, type)] = {
-        id,
-        type,
-        value: true,
-        order: order ?? this.sorter.getNewItemOrder(),
-      };
-    } else {
-      Object.assign(existing, {
-        value,
-        order: order ?? existing.order,
-      });
-    }
-  }
-
-  toggle(id: string, type: WorkspaceFavoriteItem['type']) {
-    this.set(id, type, !this.isFavorite(id, type));
-  }
-
-  remove(id: string, type: WorkspaceFavoriteItem['type']) {
-    this.adapter.ensureRootProperties();
-    const existing = this.getFavoriteItem(id, type);
-    if (existing) {
-      existing.value = false;
-    }
-  }
-
   clearAll() {
     this.adapter.cleanupFavorites();
   }
 }
 
-type CompatibleFavoriteSupportType =
-  | WorkspaceFavoriteItem['type']
-  | FavoriteSupportType;
+type CompatibleFavoriteSupportType = FavoriteSupportType;
 
 /**
- * A service written for compatibility,with the same API as FavoriteItemsAdapter.
- * When `runtimeConfig.enableNewFavorite` is false, it operates FavoriteItemsAdapter,
- * and when it is true, it operates FavoriteService.
+ * A service written for compatibility,with the same API as old FavoriteItemsAdapter.
  */
 export class CompatibleFavoriteItemsAdapter extends Service {
```

Comment:
rename `FavoriteItemsAdapter`?

Reply:
This class will be replaced by FavoriteService later